src/frontend/planner_test/tests/testdata/output/temporal_filter.yaml

```diff
@@ -383,17 +383,15 @@
       │   └─StreamDynamicFilter { predicate: ($expr1 > now), output_watermarks: [$expr1], output: [t1.ts, $expr1, t1._row_id], cleaned_by_watermark: true }
       │     ├─StreamProject { exprs: [t1.ts, AddWithTimeZone(t1.ts, '01:00:00':Interval, 'UTC':Varchar) as $expr1, t1._row_id] }
       │     │ └─StreamFilter { predicate: Not((t1.ts > '2023-12-18 00:00:00+00:00':Timestamptz)) }
-      │     │   └─StreamShare { id: 2 }
-      │     │     └─StreamFilter { predicate: (Not((t1.ts > '2023-12-18 00:00:00+00:00':Timestamptz)) OR (t1.ts > '2023-12-18 00:00:00+00:00':Timestamptz)) }
```

Comment:
Not sure if this should be changed to `IsNotNull(t1.ts)`, current implementation simply returns `True` or `False.

Reply:
As a general optimization, we should convert it into `IsNotNull` in this case, because column's data type in risingwave is nullable. However, if the expression itself is `isNotNull` and `isNull`, we can definitely convert it into true or false?

Reply:
> However, if the expression itself is isNotNull and isNull, we can definitely convert it into true or false?

Yep I think so, for `Not (e) [AND / OR] (e)`, iff `e` is `IsNotNull` or `IsNull` should we directly convert it to `True` or `False` depending on the relevant context.